### PR TITLE
Get the web side styled, and unify web+native styles as much as possible

### PR DESCRIPTION
**Pull request by legastero**
screen shot 2016-07-08 at 2 40 20 am

This adds a `createStyleSheet()` method to the base styles feature so we can define native and web styles at the same time. This will likely be replaced if we go with a library later.

This adds a `FontAwesome.css` file to the base styles feature. If there is a more suitable place, it can be moved.

### CODE REVIEW

File: features/base/styles/_.web.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export * from './web';
+
+require('style!./styles/FontAwesome.css');
```
Comment (tsareg) [inline features/base/styles/_.web.js:3]: I'm not sure about putting 3rd party stuff inside source code. What if we have it as npm module (https://www.npmjs.com/package/font-awesome) instead?

File: package.json [hunk 2]
```diff
@@ -8,7 +8,9 @@
     "eslint": "node_modules/eslint/bin/eslint.js \"config.js\" \"features/**/*.js\" \"index.*.js\"",
     "start:android": "npm run clean; react-native run-android",
     "start:ios": "npm run clean; react-native run-ios",
-    "start:web": "webpack-dev-server --inline --hot --colors --port 5000 --https --history-api-fallback --open"
+    "start:web": "webpack-dev-server --inline --hot --colors --port 5000 --https --history-api-fallback --open",
+    "lint": "eslint .",
+    "validate": "npm ls"
```
Comment (lyubomir) [inline package.json:13]: What are these 2: "lint" & "validate"?

File: webpack.config.js [hunk 3]
```diff
@@ -32,6 +32,19 @@ module.exports = {
...
+                loader: 'style-loader!css-loader',
+            },
+            {
+                test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+                loader: "url-loader?limit=10000&minetype=application/font-woff"
+            },
+            {
+                test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+                loader: "file-loader"
+            },
```
Comment (lyubomir) [inline webpack.config.js:53]: @legastero, could you please add a new commit to this PR with comments on these 3 (like I have a comment bellow)?

File: package.json (hunk 2 above)
Comment (tsareg) [inline package.json:13]: This is something that "npm install/update" adds if you have precommit-hook inside node_modules. Even if you don't reference it inside package.json. Can be safely removed.

**Comment by lyubomir** [comment]
I experienced multiple issues with this PR + modifications that I applied during my review. So I pushed this PR + my modifications to https://github.com/jitsi/jitsi-meet-react/tree/legastero-web-styles, I'm going to close this PR, and @legastero is going to try to reproduce the issues in that branch and possibly fix any issues starting from there.

**Pull request closed without merging**